test/e2e/common/node/sysctl.go

```diff
@@ -173,4 +173,56 @@ var _ = SIGDescribe("Sysctls [LinuxOnly] [NodeConformance]", func() {
 		err := e2epod.WaitForPodFailedReason(f.ClientSet, pod, "SysctlForbidden", f.Timeouts.PodStart)
 		framework.ExpectNoError(err)
 	})
+
+	/*
+	  Release: v1.23
+	  Testname: Sysctl, test sysctls supports slashes
+	  Description: Pod is created with kernel/shm_rmid_forced sysctl. Support slashes as sysctl separator. The '/' separator is also accepted in place of a '.'
+	  [LinuxOnly]: This test is marked as LinuxOnly since Windows does not support sysctls
+	*/
+	framework.ConformanceIt("should support sysctls with slashes as separator [MinimumKubeletVersion:1.23]", func() {
```

Comment:
This should not be added directly as a conformance test. Let's start with a cluster e2e and consider promoting after we get good signal from the test.

Reply:
> Let's start with a cluster e2e

I don't understand what does `cluster e2e test` mean?  Where does it need to be modified?
Do you mean change  `framework.ConformanceIt` to `ginkgo.It` or  test kind?

Reply:
I think @mengjiao-liu understands @ehashman's comment correctly.
A conformance test needs to be stable as `it is stable and runs consistently (e.g., no flakes), and has been running for at least two weeks` from https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/conformance-tests.md#conformance-test-requirements
Then we need to check the stability as `ginkgo.It` for two weeks at least, then the test can promote to a conformance test after that.

Reply:
Comment addressed. Request review again. Thanks for @ehashman @oomichi